doc/userguide/lua/libs/ssh.rst

```diff
@@ -6,6 +6,16 @@ SSH transaction details are exposes to Lua scripts with the
 
   local ssh = require("suricata.ssh")
 
+If you want to use hassh, you can either set suricata.yaml option
+``app-layer.protocols.ssh.hass`` to true, or specify it in the
+``init`` function of your lua script::
+
+  function init (args)
+    local needs = {}
+    needs["ssh.hassh"] = tostring(true)
```

Comment:
See comment: https://github.com/OISF/suricata/pull/13103#discussion_r2064779701

Reply:
Thanks, done in next version